lib/tasks/deposit.rake

```diff
@@ -1,20 +1,28 @@
 # frozen_string_literal: true
 
+# NOTE: This task is largely copypasta from the DepositStatusJob, with the logic
+#       divorced from any messaging or background job frameworks. We could
+#       consider extracting the common bits to a service?
```

Comment:
Please note this note. I'd be happy to try my hand at this in a follow-up PR. Also happy to let it sit a while.

Reply:
2 repetitions is too few to justify extracting out a class.  Sandi Metz suggested even 3 is too few.

Reply:
I strongly agree with this guideline generally. In this case, I am suspicious of logic that lives *only* in jobs or in tasks because they are so coupled to how they are invoked (`ActiveJob` and `rake` respectively), which is why I'd be happy to move that logic to a service class that can be used in both contexts.